src/multiparser.js

```diff
@@ -139,26 +149,60 @@ function fromHtmlParser2(path, options) {
           parent.attribs.lang === "ts")
       ) {
         return {
-          parser: "typescript",
-          wrap: doc => concat([hardline, doc]),
+          options: { parser: "typescript" },
+          transformDoc: doc => concat([hardline, doc]),
           text: getText(options, node)
         };
       }
 
       // Inline Styles
       if (parent.type === "style") {
         return {
-          parser: "postcss",
-          wrap: doc => concat([hardline, stripTrailingHardline(doc)]),
+          options: { parser: "postcss" },
+          transformDoc: doc => concat([hardline, stripTrailingHardline(doc)]),
           text: getText(options, node)
         };
       }
 
       break;
     }
+
+    case "attribute-value": {
+      /*
+       * Vue binding sytax: JS expressions
+       * :class="{ 'some-key': value }"
+       * v-bind:id="'list-' + id"
+       * v-if="foo && !bar"
+       */
+      if (/(^v-)|:/.test(node.name) && !/^\w+$/.test(node.value)) {
```

Comment:
you may want to also include the `@` , for things like `@click="someFunction()"`